telemetry/specs/INT.mdk

```diff
@@ -397,17 +404,20 @@ receiver to ignore the value of the Checksum field. For UDP over IPv6, there
 are specific use cases in which it is possible to assign a zero Checksum 
 (as defined in RFC 6936).
 
-If an L4 Checksum update is required, an INT source/transit switch may perform
+If an L4 Checksum update is required, an INT source/transit/sink switch may perform
 it in one of two ways:
 
 * Update the L4 Checksum field such that the new value is equal to the Checksum 
-of the new packet, after the INT-related updates, or
+of the new packet, after the INT-related updates (header additions/removals, 
+field updates), or
 
 * If the INT source indicates that Checksum-neutral updates are allowed by 
 setting an instruction bit corresponding to the Checksum Complement metadata, 
 then the INT source/transit switches may assign a value to the Checksum 
 Complement metadata which guarantees that the existing L4 Checksum is the 
-correct value of the packet after the INT-related updates.
+correct value of the packet after the INT-related updates. INT sink doesn't
+need to take any special action but just removing the Checksum complemenet
```

Comment:
We need to clarify that sink has to update the L4 checksum, it cannot just remove the checksum complement. Sink is removing bytes from the packet, hence needs to recompute checksum.

This is also related to my pull request on the checksum-ignored flag. If you don't mind, revert your changes in the section on checksum. I can take care of these edits in my PR.

Reply:
Done.